### PR TITLE
chore: Remove references to amber_artwork_visibility_unlisted as this is liv…

### PR DESCRIPTION
**Pull request by jpotts244**
…e on all environments

The type of this PR is: Chore

### Description

Remove references to `amber_artwork_visibility_unlisted` Unleash flag as this feature has been released on production

After code level removals we can archive the flag in Unleash's dashboard [here](https://unleash.artsy.net/projects/default/features/amber_artwork_visibility_unlisted)

Associated PRs:

- https://github.com/artsy/gravity/pull/17763
- https://github.com/artsy/volt/pull/7615

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by damassi — APPROVED** [review]

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPrivateArtwork.tsx [hunk 1]
```diff
@@ -35,8 +31,7 @@ export const ArtworkSidebarPrivateArtwork: React.FC<ArtworkSidebarPrivateArtwork
     artwork
   )
 
-  const isPrivateArtwork =
-    privateArtworksEnabled && data.visibilityLevel === "UNLISTED"
+  const isPrivateArtwork = data.visibilityLevel === "UNLISTED"
```
Comment (Mitch-Henson) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPrivateArtwork.tsx:34]: Is it possible to change both instances of referencing the magic string to instead use `artwork?.isUnlisted`? Thinking in the future when we potentially will have multiple "private" types which we want to hide

**Review by Mitch-Henson — DISMISSED** [review]
Small, non-blocking comment

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPrivateArtwork.tsx (hunk 1 above)
Comment (jpotts244) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPrivateArtwork.tsx:34]: Good call, Ill do a clean up pass as part of this PR

**Review by Mitch-Henson — APPROVED** [review]
Looks great!

**Pull request merged**